database/migrations/2018_12_06_122254_create_tasks_table.php

```diff
@@ -17,7 +17,10 @@ public function up()
             $table->increments('id');
             $table->integer('user_id');
             $table->boolean('is_checked')->default(false);
-            $table->string('name', 255)->default('New task');
+            $table->string('name', 64)->default('New task');
+            $table->string('desc', 255)->nullable();
+            $table->string('priority', 6)->default('medium');
```

Comment:
mozda ipak boolean i da bude true ili false a po defaultu false

Reply:
Šta misliš da stavim integer od 0 do 3 i da se radi validacija da ne pređe 3 (ili bude ispod 0)?

Reply:
aha, ako zelis da imas 3 razlicita nivoa prioritya to je ok, moze, a ideja ovde je bila samo ima porioritet i nema nista posebno ali mozes i tako implementirati onda ako zelis